core/templates/local_vector.h

```diff
@@ -245,13 +245,13 @@ class LocalVector {
 	void insert(U p_pos, T p_val) {
 		ERR_FAIL_UNSIGNED_INDEX(p_pos, count + 1);
 		if (p_pos == count) {
-			push_back(p_val);
+			push_back(std::move(p_val));
```

Comment:
Is this safe for the non `T&&` case? I'm not necessarily 100% sure it isn't but this feels like a potential error.

Reply:
You are right, all of the cv and reference qualifiers are stripped from T, I kinda brain farted on this.